pynanocoin.py

```diff
@@ -363,6 +364,14 @@ def parse_payload(cls, hdr, rawdata):
 
     @classmethod
     def make_packet(cls, peers: Iterable[Peer], net_id, version: int):
+        peers = list(peers)
+
+        if len(peers) > 8:
```

Comment:
I'd make this an assert rather than an exception.
It is not an exception, it is coding error (bug) to trying to create a keepalive with more than 8 peers.

Reply:
I'll just remove it, since there's already an assert statement in the constructor